### PR TITLE
Mention cert-manager-verifier in the docs

### PR DESCRIPTION
**Pull request by alenkacz**

### CODE REVIEW

**Comment by alenkacz** [comment]
@meyskens let me know if you want to move it to different place or word a bit differently :)

Thanks!

File: content/en/docs/installation/kubernetes.md [hunk 1]
```diff
@@ -164,6 +164,8 @@ README](https://hub.helm.sh/charts/jetstack/cert-manager).
 
 ## Verifying the installation
 
+(Optionally the whole verification flow is automated by running tool maintained by the community (cert-manager-verifier)[https://github.com/alenkacz/cert-manager-verifier])
```
Comment (munnerz) [inline content/en/docs/installation/kubernetes.md:167]: Could we maybe move this to the bottom of this section? Otherwise, lgtm! :)

**Comment by wallrj** [comment]
Hey @alenkacz 

Thanks for this!

We're releasing cert-manager 1.2 soon (next week) so if you have a moment, please test the latest alpha release with your verification tool:
 * https://github.com/jetstack/cert-manager/releases

Please address the comment by @munnerz so that we can get this merged in.

And I think `cert-manager-verifier` would make a great addition to the https://cert-manager.io/docs/usage/kubectl-plugin/ so come and join one of our standup meetings if you feel like chatting about that.

/assign @alenkacz

**Comment by alenkacz** [comment]
@wallrj thank, I fixed the position in the docs and also added tests for 1.2.0 - everything is working, yay.

Yeah I think it would be great addition to that kubectl plugin!

**Pull request merged**